### DISCUSSION TITLE
Suggestion for Glance

### DISCUSSION THREAD

**Discussion by ironcloudy** [zen-browser/desktop#2522] [General]
Just updated to a.17.  Tested the updated Glance and love the smoothness!
Some suggestions:

1. The viewport can shrink with exact pixels instead of proportionally, so the frame thickness is consistent all sides; visual consistency and more intentional.
2. Bottom and right side of the viewport has strange lines, I've highlighted in the screenshot...are those shadows? it's a little odd in a dark bg.
3. I feel the contents behind Glance can go even further, darken even more or even blur (exclude frame). Screenshot shows the distinction is subtle. Can be a little confusing in certain sites.

![Screenshot 2024-10-31 221344](https://github.com/user-attachments/assets/4d81243d-37fb-49d3-87d1-7ddfbb46e9ee)
![Screenshot 2024-10-31 223635](https://github.com/user-attachments/assets/1fd94ed8-f47b-4f39-9dbd-cac59ceb870d)

**Comment by Milkman337** [comment]
Another Suggestion:
It Would be really cool to have the option to automatically use glance for links that would open in a new tab.

**Reply by rubaboo** [reply]
I suspect most people don't expect and don't want that. It was bad enough when Glance hijacked the common "Open in a new tab" shortcut, Ctrl+Click.
